pyproject.toml

```diff
@@ -8,18 +8,19 @@ readme = "README.md"
 repository = "https://github.com/uc-cdis/dictionaryutils" 
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "~3.5 || ~3.6"
 PyYAML = "~=5.1"
-jsonschema = ">=2.5,<3.0"
+jsonschema = ">2.5,<4.0"
```

Comment:
is the change from `>=` to `>` necessary?

Reply:
oh thats not necessary but the `4.0` is.